docker-compose.yml

```diff
@@ -15,7 +16,7 @@ services:
       - "8000:8000"
     volumes:
       - "${PWD}:/app"
-    command: gunicorn pyslackersweb:app_factory --access-logfile - --bind=0.0.0.0:8000 --worker-class=aiohttp.GunicornWebWorker --reload
+    command: /bin/sh -c "./migrate.py && gunicorn pyslackersweb:app_factory --access-logfile - --bind=0.0.0.0:8000 --worker-class=aiohttp.GunicornWebWorker --reload"
```

Comment:
This is mostly a note, not anything for this PR. However we may want to do a "wait for up" script that checks that the DB is configured and listening since docker doesn't have a way to do that natively.